meta-resin-common/recipes-containers/docker/docker/docker-host.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Docker Application Container Engine
```

Comment:
Wouldn't it be nice to have the description different here? In this way it would be easier to distinguish this service from the "normal" one (https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-containers/docker/docker/docker.service#L2) .

Reply:
What do you propose? I think about it as a docker daemon, similar to how instantiated units have the same description, that's why I left it the same.

Reply:
Maybe append a "(host)"?

Reply:
done